dev-docs/RFCs/vNext/composite-layer-customization-rfc.md

```diff
@@ -29,9 +29,9 @@ There are two types of naming possible
 The `forwardBaseLayerProps()` method could check for `transferAllProps`
 
 
-### Proposal: CompositeLayers take new a `subLayerProps` prop
+### Proposal: CompositeLayers take a new `subLayerProps` prop
 
-The sublayerProps would be a recursive object. A special prop `layer` can be used to override layer type.
+The `subLayerProps` prop would be a recursive object. The keys are sublayer ids and the values are object of props to override. A special prop `type` can be used to override layer type.
```

Comment:
We use `type` to specify class constructors in both the `controller` API and the mapbox custom layers integration.

Reply:
Maybe describe what values does `type` could be?